### PR TITLE
Waits until widget DOM paint before calling `afterRender`

### PR DESCRIPTION
**Pull request by shawyu-okta**
## Description:

`afterRender` fix applied as part of https://github.com/okta/okta-signin-widget/pull/2882 for `0.0` beta branch

## PR Checklist

- [x] Have you verified the basic functionality for this change?
- [ ] Did you add tests, as appropriate, following our [Automated Test guidelines](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/2676497890/Automated+Testing+in+the+Signin+Widget)?
- [ ] Did you follow our [Security Best Practices](https://oktawiki.atlassian.net/wiki/display/eng/Security+Best+practices)?
- [ ] Did you verify the change by running [downstream monolith artifact](https://oktawiki.atlassian.net/wiki/spaces/eng/pages/102897979/Sign-in+Widget+Development#Sign-inWidgetDevelopment-Instructionstocreateandrunthedownstreamartifact(d16t))? (Provide link to build below)
- [ ] Does this PR include noticeable changes to the UI? (If yes, attach screenshots/video below)

### Issue:

- [OKTA-541518](https://oktainc.atlassian.net/browse/OKTA-541518)

### CODE REVIEW

File: src/v3/src/components/Form/Form.tsx [hunk 1]
```diff
@@ -30,10 +30,15 @@ const Form: FunctionComponent<{
...
     setMessage,
     dataSchemaRef,
+    setWidgetRendered,
   } = useWidgetContext();
   const onSubmitHandler = useOnSubmit();
   const onValidationHandler = useOnSubmitValidation();
 
+  useEffect(() => {
+    setWidgetRendered(true);
+  });
```
Comment (glenfannin-okta) [inline src/v3/src/components/Form/Form.tsx:40]: Should we consider resetting this flag after each re-render? Or in `useOnSubmit` hook when we set the new transaction? (No need to consider that now for this PR but just thinking about the future.

**Review by glenfannin-okta — APPROVED** [review]

File: src/v3/src/components/Form/Form.tsx (hunk 1 above)
Comment (shawyu-okta) [reply src/v3/src/components/Form/Form.tsx:40]: Good call, I don't think this works when you move to a second view right now. Let me investigate to see if we can make this better

**Review by leemchale-okta — APPROVED** [review]

File: src/v3/src/components/Form/Form.tsx (hunk 1 above)
Comment (lesterchoi-okta) [reply src/v3/src/components/Form/Form.tsx]: @glenfannin-okta Good callout! For parity, the `afterRender` _should_ get called after each screen is painted.

**Review by glenfannin-okta — APPROVED** [review]
Awesome!

**Pull request merged**